src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -929,6 +923,20 @@ export default {
 }
 </script>
 
+<style lang="scss">
+// Enforce NcAutoCompleteResult to have proper box-sizing
+.tribute-container {
+	position: absolute;
+	box-sizing: content-box !important;
+
+	& *,
+	& *::before,
+	& *::after {
+		box-sizing: inherit !important;
+	}
+}
+</style>
```

Comment:
Do we need to set `position: absolute;` here?
It seems that it is set by tribute as an inline style anyway.

Reply:
It set as a style rule after mounting, before that - it is static and cause flickering on the right